<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>table_module</class>
 <widget class="QWidget" name="table_module">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>387</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="sheet_bar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>371</y>
     <width>960</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(230, 230, 230);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget_sheet">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>385</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QTabBar::tab{width:52}
QTabBar::tab{height:16}
QTabBar{background-color:rgb(230, 230, 230)}
</string>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::South</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="sheet_1">
    <attribute name="title">
     <string>Sheet1</string>
    </attribute>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>60</y>
       <width>256</width>
       <height>192</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="sheet_2">
    <attribute name="title">
     <string>Sheet2</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_add_sheet">
   <property name="geometry">
    <rect>
     <x>507</x>
     <y>372</y>
     <width>13</width>
     <height>13</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(230, 230, 230);
border-image: url(C:/Users/hp/Desktop/srp/fastplot/project/main_demo/graph/add_sheet.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QScrollBar" name="horizontalScrollBar">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>370</y>
     <width>160</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
